### PR TITLE
Release/4.6.1

### PR DESCRIPTION
**Pull request by NachoSoto**
### Bug fixes

* `EntitlementInfo.isActive` returns true if `requestDate == expirationDate` (#1684) via beylmk (@beylmk)
* Fixed usages of `seealso` (#1689) via NachoSoto (@NachoSoto)
* Fixed `ROT13.string` thread-safety (#1686) via NachoSoto (@NachoSoto)
* `PurchasesOrchestrator`: replaced calls to `syncPurchases` with posting receipt for an individual product during SK2 purchases (#1666) via NachoSoto (@NachoSoto)

### CODE REVIEW

**Comment by NachoSoto** [comment]
Lots of nice bug fixes in this one.

File: Tests/InstallationTests/CommonFiles/RevenueCat-Swift.h [hunk 1]
```diff
@@ -568,21 +568,33 @@ SWIFT_CLASS_NAMED("EntitlementInfos")
 /// Dictionary of active <code>EntitlementInfo</code> objects keyed by their identifiers.
 /// warning:
 /// this is equivalent to <code>activeInAnyEnvironment</code>
-/// seealso:
-/// <code>activeInCurrentEnvironment</code>
+/// <h4>Related Symbols</h4>
```
Comment (beylmk) [inline Tests/InstallationTests/CommonFiles/RevenueCat-Swift.h:571]: doesn't really matter but i'm assuming this means main isn't fully merged into this branch..?

**Review by beylmk — APPROVED** [review]
🥳

File: Tests/InstallationTests/CommonFiles/RevenueCat-Swift.h (hunk 1 above)
Comment (NachoSoto) [reply Tests/InstallationTests/CommonFiles/RevenueCat-Swift.h:571]: Hmm what do you mean? This should have everything that's in `main`. Did I miss something?

Comment (beylmk) [reply Tests/InstallationTests/CommonFiles/RevenueCat-Swift.h:571]: oh or the seealso fix went into this branch** nope all good

Comment (NachoSoto) [reply Tests/InstallationTests/CommonFiles/RevenueCat-Swift.h:571]: Yup!

**Pull request merged**